repositories01.list

```diff
@@ -4,6 +4,8 @@ https://github.com/TGAC/earlham-galaxytools
 https://github.com/AAFC-MBB/Galaxy
 https://github.com/phac-nml/galaxy_tools
 https://github.com/workflow4metabolomics/tools-metabolomics
+https://github.com/workflow4metabolomics/metaMS
+https://github.com/workflow4metabolomics/tool-bank-hmdb/
```

Comment:
I think the consensus was to move all tools under https://github.com/workflow4metabolomics/tools-metabolomics/ to consolidate reviews and CI integration.

Reply:
Let's see, I opened upstream issues.